util/testutil/integration/dockerd.go

```diff
@@ -197,21 +223,112 @@ func waitForAPI(ctx context.Context, apiClient *client.Client, d time.Duration)
 	return nil
 }
 
+const (
+	DockerdSkipCacheExport      = "cache export"
+	DockerdSkipCacheImport      = "cache import"
+	DockerdSkipDirectPush       = "direct push"
+	DockerdSkipFrontendOutline  = "frontend.outline"
+	DockerdSkipFrontendTargets  = "frontend.targets"
+	DockerdSkipImageExporter    = "image exporter"
+	DockerdSkipInfo             = "info"
+	DockerdSkipMultiCacheExport = "multi cache export"
+	DockerdSkipMultiPlatform    = "multi-platform"
+	DockerdSkipOCIExporter      = "oci exporter"
+	DockerdSkipSecurityMode     = "security mode"
+	DockerdSkipSourceDateEpoch  = "source date epoch"
+)
+
+var (
+	dockerdMobyFeaturesEnabled = map[string]bool{
+		DockerdSkipCacheExport:      false,
+		DockerdSkipCacheImport:      false,
+		DockerdSkipDirectPush:       false,
+		DockerdSkipFrontendOutline:  false,
```

Comment:
outline/target/info/epoch tests should always pass. Maybe you are testing against an old commit.

Reply:
Agree they should pass if implemented. I made tests against 20.10, master and https://github.com/rumpl/moby/tree/c8d.